### PR TITLE
Add TO Client API for Servercapabilityautomation Automation

### PR DESCRIPTION
**Pull request by dmohan001c**
## What does this PR (Pull Request) do?

- [x] This PR is not related to any Issue

## Which Traffic Control components are affected by this PR?

- CDN in a Box
- Traffic Control Client 
- Traffic Ops
- CI tests

## What is the best way to verify this PR?
Execute all the Integration tests and make sure the tests are passed.

## The following criteria are ALL met by this PR

- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: traffic_ops/testing/api/v4/servercapabilities_test.go [hunk 1]
```diff
@@ -36,13 +36,17 @@ func TestServerCapabilities(t *testing.T) {
...
+		CreateTestServerCapabilityAlreadyExist(t)
+		GetTestServerCapabilitiesByInvalidName(t)
 		UpdateTestServerCapabilities(t)
 		UpdateTestServerCapabilitiesWithHeaders(t, header)
 		header = make(map[string][]string)
 		etag := rfc.ETag(currentTime)
 		header.Set(rfc.IfMatch, etag)
 		UpdateTestServerCapabilitiesWithHeaders(t, header)
 		ValidationTestServerCapabilities(t)
+		UpdateTestServerCapabilititesInvalidData(t)
```
Comment (rimashah25) [inline traffic_ops/testing/api/v4/servercapabilities_test.go:48]: Spelling mistake in `Capabilities`

**Comment by rimashah25** [comment]
Code has been correctly refactored to TO client and the coverage is good too.
Once the latest commit was merged to this branch, integration/jobs tests pass but integration/DeliveryServices tests still fail on local.

File: traffic_ops/testing/api/v4/servercapabilities_test.go (hunk 1 above)
Comment (dmohan001c) [reply traffic_ops/testing/api/v4/servercapabilities_test.go:48]: Fixed.

**Comment by rimashah25** [comment]
Everything LGTM.

**Review by rimashah25 — APPROVED** [review]

**Pull request merged**